packages/dapp/src/providers/I18nProvider.tsx

```diff
@@ -5,10 +5,19 @@ import React, { PropsWithChildren, useMemo } from 'react';
 import { I18nextProvider, initReactI18next } from 'react-i18next';
 import { useSettingsStore } from '../stores/settings';
 import resourcesToBackend from 'i18next-resources-to-backend';
+import en from '../i18n/en/translation.json';
 
 export const I18NProvider: React.FC<PropsWithChildren<{}>> = ({ children }) => {
   const languageMode = useSettingsStore((state) => state.languageMode);
 
+  const resources = {
+    en: {
+      translation: {
+        ...en,
+      },
+    },
+  };
```

Comment:
I think this works fine as well

```ts
const resources = {
    en: {
      translation: en,
    },
  };
```

Or we can even make it inline in the init function below